### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by Umuhan**
### Below are the changes made to the code:

- [x] Deleted all the boilerplate.
- [x] Created a new directory called components.
- [x] Inside components, created a new Calculator.js file.
- [x] In Calculator.js, created a React component that matches this design:

### CODE REVIEW

File: src/components/Button.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <button type="button" className="color">
+      <span>x</span>
+    </button>
+    <button type="button">4</button>
+    <button type="button">5</button>
+    <button type="button">6</button>
+    <button type="button" className="color">
+      -
+    </button>
+    <button type="button">1</button>
```
Comment (Ibizugbe) [inline src/components/Button.js:26]: Nice job so far, Nice job creating components for buttons👍🏼. Probably this was meant to be a child component that should be used in the calculator component but I noticed you are not making use of it anywhere😉.  But just as required, the calculator component should have a child component. Kindly break the calculator component into at least one child component. apart from this being a requirement, it is important for the separation of concerns and making our code base easily maintainable😉.


image

**Review by Ibizugbe — CHANGES_REQUESTED** [review]
Hi @Umuhan 👋🏼👋🏼👋🏼, 

require_changes


Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights✅✅✅

- [x] All boilerplate code, images and texts has been taken away👍🏼.
- [x] A component directory has been created as required👍🏼.
- [x] A calculator functional component has been created👍🏼.


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js [hunk 2]
```diff
@@ -1,114 +1,12 @@
-import React, { useState } from 'react';
-import calculate from '../logic/calculate';
+import React from 'react';
+import Button from './Button';
 import '../styles.css';
 
 function Calculator() {
```
Comment (shahadat3669) [inline src/components/Calculator.js:5]: - [ ] [Optional]👍 You've done a great job by using arrow functions in most of your functional components; they're more concise and aligned with modern JavaScript practices. However, in the `Calculator` component, you've used the old function component. I recommend using arrow functions consistently throughout your project, as they are a more modern and cleaner approach. Keep up the good work! 👏🚀📝

**Review by shahadat3669 — APPROVED** [review]
Hi @Umuhan, 

Your project is complete!  There is nothing else to say other than it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ All the boilerplate that got created by the CRA (texts, images, and styles) are deleted.
✅ The `Calculator.js` component is a **functional** component
✅ `Calculator.js` have 1 child component

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

## Have any doubt ❓

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

<table  width="100%" align="center">
  <tr>
    <td colspan="4" align="center">
      <strong>Contact me: Shahadat Hossain</strong>
    </td>
  </tr>
  <tr>
    <td align="center">
      <a href="https://github.com/shahadata3669">
        GitHub
      </a>
    </td>
    <td align="center">
      <a href="https://www.linkedin.com/in/shahadat-cseng/">
        LinkedIn
      </a>
    </td>
    <td align="center">
      <a href="https://app.slack.com/client/T47CT8XPG/D05GT4E4WAY/rimeto_profile/U04426SD71V">
        Slack
      </a>
    </td>
    <td align="center">
      <a href="mailto:shahadat3669@gmail.com?subject=Microverse%20Code%20Review">
        Email
      </a>
    </td>
  </tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**